gematria/datasets/pipelines/compile_modules_lib_test.py

```diff
@@ -190,6 +192,16 @@ def test_get_bbs(self):
     self.assertLen(block_hex_values, 2)
     self.assertContainsSubset(['B801000000', 'B802000000'], block_hex_values)
 
+    with open(
+        vocab_output_file_pattern + '-00000-of-00001'
+    ) as vocab_file_handle:
+      vocab_tokens = [token.strip() for token in vocab_file_handle.readlines()]
+
+    self.assertLen(vocab_tokens, 4)
+    self.assertContainsSubset(
+        ['_D_', '_IMMEDIATE_', 'MOV', 'EAX'], vocab_tokens
+    )
```

Comment:
`self.assertCountEqual()` for a single assertion that does both (and has better error messages).